packages/iguazu-holocron/src/configureIguazuSSR.js

```diff
@@ -15,8 +15,11 @@
 import { iguazuReduce } from 'iguazu';
 
 export default function configureIguazuSSR({ store, module, ownProps }) {
-  return iguazuReduce(module.loadDataAsProps)({
-    store,
-    ownProps,
-  }).promise;
+  if (!global.BROWSER) {
+    return iguazuReduce(module.loadDataAsProps)({
+      store,
+      ownProps,
+    }).promise;
+  }
+  return Promise.resolve();
```

Comment:
We only need to have this function on the serverside. The `connectAsync` HoC will take care of clientside calls.